app/src/main/java/com/technopark/hw_1/NumberModel.java

```diff
@@ -0,0 +1,18 @@
+package com.technopark.hw_1;
+
+import android.graphics.Color;
+
+public class NumberModel {
```

Comment:
It's definitely great idea to create such model! But I suggest to remove second constructor with color calculation. Such models should be as stupid as possible and only keep data. Let's calculate color in another place